styles/globals.css

```diff
@@ -11,30 +11,34 @@ html {
 
 @layer base {
   body {
-    @apply bg-[#61E176] text-[#050505] tracking-wide;
+    @apply bg-[#61E176] text-[#050505] tracking-wide
   }
 
   h1, h2, h3, h4, h5, h6 {
-    @apply font-medium text-center;
+    @apply font-bold
   }
 
   h1 {
-    @apply text-4xl sm:text-6xl md:text-6xl font-[Raleway] text-left;
+    @apply text-4xl sm:text-6xl md:text-6xl font-[Raleway] text-left
   }
 
   h2 {
-    @apply text-3xl sm:text-4xl text-center;
+    @apply text-3xl sm:text-4xl text-center
+  }
+
+  h3 {
+    @apply sm:text-3xl text-center
   }
```

Comment:
Notice most styles in this page don't have semicolon (`;`) at the end of each rule